ods_ci/tests/Tests/650__distributed_workloads/test-distributed-workloads-metrics-ui.robot

```diff
@@ -167,3 +236,149 @@ Cleanup Kueue Resources
     IF    ${result.rc} != 0
         FAIL    Failed to delete kueue resources
     END
+
+Check Requested Resources
+    [Documentation]    checks requested resource contents
+    [Arguments]    ${project_name}   ${cpu_shared_quota}    ${memory_shared_quota}    ${cpu_requested}    ${memory_requested}    ${workload_type}
+    Check Expected String Equals    //*[@id="requested-resources-chart-CPU-ChartLegend-ChartLabel-0"]    Requested by ${project_name}: ${cpu_requested}
+    Check Expected String Equals    //*[@id="requested-resources-chart-CPU-ChartLegend-ChartLabel-2"]    Total shared quota: ${cpu_shared_quota}
+    Check Expected String Equals    //*[@id="requested-resources-chart-CPU-ChartLegend-ChartLabel-1"]    Requested by all projects: ${cpu_requested}
+    IF    "${workload_type}" == "Job"
+        ${memory_requested_round} =    Evaluate    int(round(${memory_requested}))
+    ELSE
+        ${memory_requested_round} =    Evaluate    round( ${memory_requested}, 1)
+    END
```

Comment:
Here i adjusted based on the resource requested value for eg for kueue workload we are requesting value is 1.978 and its rounding to 2 but in case of ray 2.794 its rounding to 2.8

Reply:
Increased kueue workloads with annotation `kueue.x-k8s.io/job-min-parallelism: "1"` and kept unique logic here

Reply:
It is bit surprsing that the backend logic of rounding off varies for kueue and ray workloads. 
Have you checked this scenarion with different set of values ?

Reply:
Logic here is unique only , its rounding to the nearest value for 2.794 to 2.8 and 1.978 to 2 , in order to validate in UI unique way i adjusted this to fetch 2 digit number like 2.8 or 3.9 something like that now